rhel7/profiles/ccc.profile

```diff
@@ -0,0 +1,271 @@
+#    - audit_rules_etc_gshadow_open
+#    - audit_rules_privileged_commands_unix_chkpwd
+#    - audit_rules_privileged_commands_userhelper
+#    - audit_rules_privileged_commands_usernetctl
+#    - audit_rules_execution_seunshare
+#    - audit_rules_privileged_commands_mount
+#    - audit_rules_privileged_commands_newgrp
+#    - audit_rules_privileged_commands_newuidmap
+#    - audit_rules_privileged_commands_gpasswd
+#    - audit_rules_privileged_commands_newgidmap
+#    - audit_rules_privileged_commands_umount
+#    - audit_rules_privileged_commands_passwd
+#    - audit_rules_privileged_commands_crontab
+#    - audit_rules_mac_modification
+#    - audit_rules_session_events
+#    - audit_rules_privileged_commands_at
+    - directory_access_var_log_audit
+#    - audit_rules_kernel_module_loading_insmod
+#    - audit_rules_kernel_module_loading_rmmod
+#    - audit_rules_kernel_module_loading_modprobe
+#    - audit_rules_kernel_module_loading_init
+#    - audit_rules_kernel_module_loading_finit
+#    - audit_rules_kernel_module_loading_delete
+
+    # Configuration (all are defaults)
+#    - var_auditd_data_retention=incremental_async  # typo? var_auditd_flush used instead
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+
+    # audispd plugins
+    - auditd_audispd_syslog_plugin_activated
+
+    ### Services
+
+    # sshd
+    - sshd_disable_root_login
+    - sshd_enable_strictmodes
+    - disable_host_auth
+    - sshd_disable_user_known_hosts
+    - sshd_disable_rhosts
+    - sshd_disable_empty_passwords
+    - sshd_disable_kerb_auth
+    - sshd_disable_gssapi_auth
+    - sshd_idle_timeout_value=10_minutes
+#    - sshd_set_idle_timeout  # duplicate, see below
+    - sshd_set_keepalive
+    - sshd_enable_warning_banner
+    - sshd_disable_rhosts_rsa
+    - sshd_use_approved_ciphers
+    - sshd_use_approved_macs
+
+    # ssh client settings
+
+    # rpcbind
+    - service_rpcbind_disabled
+
+    # Firewalld
+    - service_firewalld_enabled
+    - set_firewalld_default_zone
+
+    # abrt
+    - package_abrt_removed
+
+    # Chrony
+
+    # rngd
+
+    # SSSD
+
+    # Crypto settings
+
+    # Libreswan
+
+    ### User sessions
+
+    # Login
+    - no_empty_passwords
+    - var_accounts_passwords_pam_faillock_deny=3
+    - accounts_passwords_pam_faillock_deny
+    - var_accounts_passwords_pam_faillock_fail_interval=900
+    - accounts_passwords_pam_faillock_interval
+    - var_accounts_passwords_pam_faillock_unlock_time=never
+    - accounts_passwords_pam_faillock_unlock_time
+    - dconf_gnome_banner_enabled
+    - login_banner_text=usgcb_default
+    - dconf_gnome_login_banner_text
+    - disable_users_coredumps
+    - var_accounts_max_concurrent_login_sessions=10
+    - accounts_max_concurrent_login_sessions
+    - securetty_root_login_console_only
+    - var_password_pam_unix_remember=5
+    - accounts_password_pam_unix_remember
+
+    # Password
+#    - no_empty_passwords   # duplicate in Login
```

Comment:
I think you can drop this comment.
Unless you would like to keep track of this for some reason.